__tests__/Seq.ts

```diff
@@ -10,6 +10,14 @@
 import { isCollection, isIndexed, Seq } from '../';
 
 describe('Seq', () => {
+  it('has a default first value', () => {
+    expect(Seq().first({})).toEqual({});
+  });
+
+  it('has a default first value', () => {
+    expect(Seq().last({})).toEqual({});
+  });
```

Comment:
Should probably also have a test that ensures that calling `first` and `last` on an empty `Seq` with no default set will return `undefined`. Only to make sure the behavior is consistent with the past.